src/components/Home/overviewData/LatestTreasuryOverview.tsx

```
@@ -131,38 +132,88 @@ const LatestTreasuryOverview = ({ currentTokenPrice, available, priceWeeklyChang
-										{totalTreasuryValueUSD && (
-											<div className='flex items-baseline'>
-												<span className={`${dmSans.className} ${dmSans.variable} text-xl font-semibold text-blue-light-high dark:text-blue-dark-high`}>
-													~${totalTreasuryValueUSD}
-												</span>
+										<div className={`${dmSans.className} ${dmSans.variable} flex items-baseline gap-x-1 self-end`}>
+											<span className={' flex text-xs font-normal leading-5 text-lightBlue dark:text-blue-dark-medium'}>{chainProperties[network]?.tokenSymbol} Price</span>
+											<div className='flex items-center gap-x-1 text-lg font-semibold'>
+												<div>
+													{currentTokenPrice.value === 'N/A' ? (
+														<span className=' text-bodyBlue dark:text-blue-dark-high'>N/A</span>
+													) : currentTokenPrice.value && !isNaN(Number(currentTokenPrice.value)) ? (
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)